modules/associate.js

```diff
@@ -33,10 +33,31 @@ exports.toggleAssociation = function(thread, hook, text) {
 	return true;
 };
 
+exports.printAssociations = function(api, event) {
+	var assoc = this.config[event.thread_id];
+	var message = '';
+	for (var a in assoc) {
+		message += a + ' → ' + assoc[a] + '\n';
```

Comment:
Won't we end up with a trailing newline? Maybe {
    if (message.length > 0) { message += "/n"; }
    message += //thing
}